test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -34,6 +34,15 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, 1, Integer.valueOf(1));
         assertNotUnique(null, null);
         assertNotUnique(null, "a", "b", null);
+        
+        // some empty objects
```

Comment:
Unrelated code changes? Please refrain for adding unrelated code changes to your future PRs